packages/cadl-ranch-api/src/types.ts

```diff
@@ -52,3 +52,11 @@ export interface MockResponseBody {
   contentType: string;
   rawContent: string | undefined;
 }
+
+export enum CollectionFormat {
```

Comment:
can we leave this as a union and not an enum. string enums are often just more annoying to deal with in TS.

Reply:
okay, changed into union.